dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md

```diff
@@ -10,6 +10,9 @@ Get a tracked entity with given UID.
 
 ### `getTrackedEntityByUid.parameter.program`
 
+Get tracked entities enrolled in the specified `program`.
+When a `program` is specified, all the Tracked Entity attributes and the Program specific ones are included.
```

Comment:
I think we should be more explicit than "are included". The question I have is where are they included? This leads to the one thing I would like to know: should the TEI only have tracked entity type attributes in the response? while the enrollment should only have program attribute values? (@stian-sandvold @Bekkalizer)

The implementation is really concerning to me:
`Enrollment.trackedEntity.trackedEntityAttributeValues` should only contain the attribute values for program attributes AFAIK. On the other hand the `Enrollment.trackedEntity` is the enrolled TEI. So the TEI might have its own attribute values for tracked entity type attributes. So by setting the `Enrollment.trackedEntity.trackedEntityAttributeValues` to a subset we risk of removing the attribute values of the TEI (when hibernate is used) 😬 To me there seems to be a high risk that TEI and enrollment will show the wrong attributes.